lib/backend/resources/static.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v3 "github.com/projectcalico/libcalico-go/lib/apis/v3"
+	"github.com/projectcalico/libcalico-go/lib/backend/model"
+)
+
+// This file contains helpers for creating hard-coded resources used in the Kubernetes backend.
+const (
+	// AllowProfileName is the name of an "allow-all" profile.
+	AllowProfileName = "projectcalico-allow-all"
```

Comment:
I wonder if there's a good prefix we could use for this. We use `kns.` and `ksa.` already. A prefix could be a nice indicator that this profile was from Calico.

Reply:
We usually use the `projectcalico.org/` namespace for labels, but I don't think that `/` is allowed in resource names.

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names

So, I think what you have is probably as good as we can get.